src/TimeSteppers/runge_kutta_3.jl

```diff
@@ -135,10 +137,17 @@ function time_step!(model::AbstractModel{<:RungeKutta3TimeStepper}, Δt; callbac
     rk3_substep!(model, Δt, γ³, ζ³)
 
     tick!(model.clock, third_stage_Δt)
+    model.clock.last_stage_Δt = corrected_third_stage_Δt
```

Comment:
Don't you need to compute `corrected_third_stage_Δt` before you use it here?

Reply:
yeah I fixed this in the commit after the main merge